### ISSUE TITLE
LAMMPS parsing failure

### ISSUE THREAD

**Issue by fxcoudert** [nomad-coe/atomistic-parsers#52]
Data: https://nomad-lab.eu/prod/v1/gui/user/uploads/upload/id/3r7neW3GSU63WbJGYfZhuw
Failed to parse 2 output files, and did not recognise the trajectory.

**Comment by Pepe-Marquez** [comment]
Hey @JFRudzinski, can you have a look at this? 
Would reprocessing in staging work? 
Some screenshots in this Twitter thread just in case it provides more info: https://twitter.com/fxcoudert/status/1680665798703030272?s=20

**Comment by JFRudzinski** [comment]
@fxcoudert I am sorry for the VERY long delay in getting to this issue!!

I am trying to dedicate some time now to improving the Lammps parser. If you wouldn't mind sharing your data that failed to parse with me, it would be very helpful. If the above-mentioned upload exist, you can give me access by adding me as a reviewer via the "manage upload members" button (icon looks like 2 people 
![Screenshot from 2024-01-04 10-29-30](https://github.com/nomad-coe/atomistic-parsers/assets/11678418/bc5850a3-6cb1-4a3c-8fd9-2b6e649e8994)
) on the upload page in NOMAD.

**Comment by fxcoudert** [comment]
You are now added as reviewer for this upload:

<img width="569" alt="Capture d’écran 2024-01-04 à 15 42 27" src="https://github.com/nomad-coe/atomistic-parsers/assets/1980544/6403d8aa-de49-45a1-b6f6-b914020d1173">

**Comment by JFRudzinski** [comment]
@fxcoudert excellent, thank you very much!

**Comment by JFRudzinski** [comment]
This issue appears to be linked to #37 , will confirm after solving that one

**Comment by JFRudzinski** [comment]
Update: The major processing issues here have now been solved. This had to do with:
1. Not recognizing the data file
2. Nan calculated for mass fraction for systems without recognized atoms

Once these changes go through, the system and method data will be properly parsed. However, overall workflow within this example is not recognized (i.e., multiple MD runs). This corresponds to a bigger extension of Lammps support needed. I have opened an issue for this #89 if anyone would like to follow the progress
